client/upe-onboarding-wizard/upe-preview-methods-selector/add-payment-methods-task.js

```diff
@@ -20,6 +21,16 @@ import PaymentMethodCheckboxes from '../../components/payment-methods-checkboxes
 import PaymentMethodCheckbox from '../../components/payment-methods-checkboxes/payment-method-checkbox';
 import LoadableSettingsSection from 'wcstripe/settings/loadable-settings-section';
 
+const HeadingWrapper = styled.div`
+	display: flex;
+	margin-bottom: 1em;
+	gap: 8px;
```

Comment:
Is this rule outdated now?

```suggestion
```

Reply:
Without it, the items are a bit too close:
![2021-10-01 08 14 26](https://user-images.githubusercontent.com/273592/135625778-431f2e78-6f53-44e1-a80b-f19aebac386f.gif)

We're making pretty extensive use of `gap` in the styles of this plugin, that's one of the reasons why I thought it would be ok to use it here too 😅 
In which sense is it outdated? 👀

Reply:
Thanks for the explanation! Ah, I was looking if there are any spacing changes between the heading and the list (the link was probably hidden at the time) and figured since that element is using flexbox, the grid `gap` rule might be something left over from previous work. It clearly is not! My bad!